Form/DoctrineMongoDBTypeGuesser.php

```diff
@@ -165,13 +150,8 @@ public function guessMinLength($class, $property)
     {
     }
 
-    /**
-     * @param string $class
-     * @param string $property
-     *
-     * @return ValueGuess|null
-     */
-    public function guessPattern($class, $property)
+    /** @return ValueGuess|null */
+    public function guessPattern(string $class, string $property)
```

Comment:
[`FormTypeGuesserInterface::guessPattern()` already has `string` in 5.4](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Form/FormTypeGuesserInterface.php#L52)